deploy/backplane/10-cee.ClusterRole.yml

```
@@ -0,0 +1,18 @@
+aggregationRule:
+  clusterRoleSelectors:
+    # aggregate all "view" scope rbac
+    - matchExpressions:
+        - key: rbac.authorization.k8s.io/aggregate-to-view
+          operator: In
+          values:
+            - "true"
+    - matchExpressions:
+        - key: managed.openshift.io/aggregate-to-backplane-cee
+          operator: In
+          values:
+            - "true"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-cee
```

Been thinking about names and consistency.  If we borrow from existing patterns we do something like: {user group}-{admins,readers}-{project,cluster}.  In that case:

```suggestion
  name: backplane-cee-readers-cluster
```